js/classes/entities/midground/npc/village/papa_chad.js

```diff
@@ -5,7 +5,11 @@
  * @author Devin, Caleb, Nathan, Trae
  */
 class PapaChad {
-    constructor(pos) {
+    /**
+     * @param {Vector} pos the position at which he should spawn. 
+     * @param {Conversation} convo The conversation that will show if PapaChad is interacted with.
+     */
+    constructor(pos, convo = null) {
```

Comment:
Good call passing in `convo` as a parameter instead of hardcoding the converation(s) into the class itself.

Reply:
Yeah, I mean depending on our story progress his convo would be different so we gotta be able to determine that.